packages/strapi-plugin-content-manager/admin/src/components/InputUID/index.js

```diff
@@ -115,13 +115,6 @@ const InputUID = ({
     }
   };
 
-  useEffect(() => {
-    if (!value && validations.required) {
-      generateUid.current(true);
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
```

Comment:
I don't understand why this is removed? The UID has to be generated if the field is required.

Reply:
Because when updating a document the field gets regenerated, this solves the issue.